### ISSUE TITLE
yoast redirects node not exist in wpGraphQL

### ISSUE THREAD

**Issue by zmn-aamir** [ashhitch/wp-graphql-yoast-seo#38]

**Comment by izzygld** [comment]
@ashhitch we should probably add a feature for "premium" version of Yoast.

**Comment by zmn-aamir** [comment]
we are already using Yoast premium version, but there is no option for getting redirects data in wp-graphQL query.

**Comment by ashhitch** [comment]
Would be great to get the premium features into the plugin.

I don't currently have a premium license for any of the sites I manage.

Will do some research into all the extras.

**Comment by ashhitch** [comment]
I now have access to the premium version so can start to investigate this further

**Comment by ashhitch** [comment]
@zmn-aamir redirects are now available from #44 
You will need version v4.3.0

```
query MyQuery {
  seo {
    redirects {
      type
      target
      origin
      format
    }
  }
}
```
